templates/cmake/apple.cmake

```diff
@@ -8,10 +8,12 @@ macro(cc_ios_before_target target_name)
     list(APPEND CC_PROJ_SOURCES
         ${CC_UI_RESOURCES}
     ) 
-    if (USE_PORTRAIT)
-        configure_file(${CC_PROJECT_DIR}/LaunchScreenBackgroundPortrait.png ${CC_PROJECT_DIR}/LaunchScreenBackground.png COPYONLY)
-    else()
-        configure_file(${CC_PROJECT_DIR}/LaunchScreenBackgroundLandscape.png ${CC_PROJECT_DIR}/LaunchScreenBackground.png COPYONLY)
+    if(NOT EXISTS ${CC_PROJECT_DIR}/LaunchScreenBackground.png)
+        if (USE_PORTRAIT)
+            configure_file(${CC_PROJECT_DIR}/LaunchScreenBackgroundPortrait.png ${CC_PROJECT_DIR}/LaunchScreenBackground.png COPYONLY)
+        else()
+            configure_file(${CC_PROJECT_DIR}/LaunchScreenBackgroundLandscape.png ${CC_PROJECT_DIR}/LaunchScreenBackground.png COPYONLY)
+        endif()
```

Comment:
Can current editor support change orientation after first build?

Reply:
No, modification is disabled.